hangman/game.py

```diff
@@ -1,23 +1,76 @@
 from .exceptions import *
+from random import randint
 
 # Complete with your own, just for fun :)
-LIST_OF_WORDS = []
+LIST_OF_WORDS = ['Python', 'Hello World', 'Women Who Code']
 
 
 def _get_random_word(list_of_words):
     pass
+    if len(list_of_words) != 0:
+       random_index = randint(0, len(list_of_words)-1)
+    else:
+        raise InvalidListOfWordsException()
+    return list_of_words[random_index];
 
 
 def _mask_word(word):
     pass
+    if word != '':
+        letters_of_word = list(word)
+        for letter in letters_of_word:
+            word = word.replace(letter, '*')
+        return word
+    else:
+        raise InvalidWordException()
 
 
 def _uncover_word(answer_word, masked_word, character):
     pass
-
-
+    if len(answer_word) == 0:
+      raise InvalidWordException()
+    elif len(character) > 1:
+      raise InvalidGuessedLetterException()
+    elif len(answer_word) != len(masked_word):
+      raise InvalidWordException()
+    else:
+      answer_word = answer_word.lower()
+      characters = list(answer_word)
+      masked_word_list = list(masked_word)
+      counter = 0
+      for c in characters:
+        character = character.lower()
+        if c == character:
+          masked_word_list[counter] = character
+        counter = counter + 1
+      return ''.join(masked_word_list)
+    
+    
 def guess_letter(game, letter):
     pass
+    if game['remaining_misses'] == 0 or game['answer_word'].lower() == game['masked_word'].lower():
+      raise GameFinishedException
+    if len(list(letter)) > 1:
```

Comment:
`if len(letter) > 1:`

again, you don't need to convert to list ... everything a collection can do, `string` should be able to do as well.. cause `string` is a type of collection too ...